applications/nrf_desktop/README.rst

```diff
@@ -62,143 +62,187 @@ The project contains configuration files for following boards:
+No additional software or drivers are required.
 
-BLE Peer control
-----------------
+The devices that are connected both wirelessly and through the USB at the same time will provide the input only through the USB.
+After the USB disconnection, the device will automatically switch to sending the data wirelessly using Bluetooth.
 
-The connected BLE peer can be controlled using predefined button or button combination:
+Battery-powered devices with rechargeable batteries are charged through the USB.
 
-* gaming mouse - "aim button" located on the left side of the mouse.
-* desktop mice - press the scroll (the scroll has to be pressed before the mouse is powered up).
-* keyboard - Page Down key pressed together with Fn modifier key.
-* dongle - the "SW1" button.
+Connection through BLE
+======================
 
-Gaming mouse also has dedicated hardware switch located at the bottom.
-The switch is used to select between:
+When turned on, the nRF Desktop peripherals are advertising until they go to the suspended state or connect through Bluetooth.
+The peripheral supports one wireless connection at a time, but it may be bonded with multiple peers.
 
-* dongle peer - top position.
-* other BLE peers - middle position.
-* mouse turned off - bottom position.
+The nRF Desktop central scans for all bonded peripherals that are not connected.
+The scanning is interrupted when any device connected to the dongle through Bluetooth is in use.
+Continuing the scanning in such scenario would cause report rate drop.
 
-The following peer control operations are available:
+The scanning for new peers starts automatically when one of the bonded peers disconnects.
+It also takes place periodically when a known peer is not connected.
 
-* **peer selection**
 
-  - Peripheral supporting multiple peers (the gaming mouse, the keyboard)
+BLE peer control
+----------------
 
-    Short click to initialize peer selection.
-    During the peer selection short click selects the next peer and double click confirms the selection.
-    The peer is changed after the confirmation.
+A connected BLE peer device can be controlled using predefined buttons or button combinations.
+There are several peer operations available.
 
-    For the gaming mouse the dedicated hardware switch is used to select the dongle peer.
-    When the dongle peer is selected, the peer control is disabled until the switch will be set to another position.
+The peer operation states provide visual feedback through LEDs (if the device has LEDs).
+Each of the states is represented by separate LED color and effect.
 
-* **bond erase**
+Gaming mouse
+++++++++++++
+
+The following predefined buttons or button combinations are assigned to peer control operations for the gaming mouse:
+
+Hardware switch:
+    * The switch is located next to the optical sensor.
+    * You can set the switch in the following positions:
+
+        * Top position: Connection to the dongle peer active.
```

Comment:
Maybe better: Dongle peer is selected ?
If dongle will not establish the connection, there will be no active connection.

Reply:
That's ok, as long as you keep the style and format of other bullet points consistent.

Reply:
done